buildtools/ci.template.yml

```diff
@@ -0,0 +1,164 @@
+Parameters:
```

Comment:
Like discussed, this is something we want 
- shared space to avoid duplication
- automatic deployment of the stack on code push